src/app/login/components/appVersionMessageBar.tsx

```diff
@@ -0,0 +1,57 @@
+/***********************************************************
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License
+ **********************************************************/
+import * as React from 'react';
+import { MessageBar, MessageBarType } from 'office-ui-fabric-react/lib/MessageBar';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+import { LocalizationContextConsumer, LocalizationContextInterface } from '../../shared/contexts/localizationContext';
+import { ResourceKeys } from '../../../localization/resourceKeys';
+import { fetchLatestReleaseTagName, latestReleaseUrlPath } from '../../api/services/githubService';
+import * as packageJson from '../../../../package.json';
+import { isNewReleaseVersionHigher } from './appVersionHelper';
+
+interface AppVersionMessageBarState {
+    latestReleaseVersion?: string;
+}
+
+export default class AppVersionMessageBar extends React.Component<{}, AppVersionMessageBarState> {
+
+    public render() {
+        return (
+            <LocalizationContextConsumer>
+                {(context: LocalizationContextInterface) => (
+                    this.renderMessageBar(context)
+                )}
+            </LocalizationContextConsumer>
+        );
+    }
+
+    public componentDidMount() {
+        fetchLatestReleaseTagName().then(tagName => {
+            this.setState({latestReleaseVersion: tagName});
+        });
+    }
+
+    private readonly hasNewerRelease = () => {
+        if (!this.state || !this.state.latestReleaseVersion) { return false; }
+        const latestVersion = this.state.latestReleaseVersion;
+        const versionTagPrefix = 'v';
+        const semanticVersion = latestVersion.startsWith(versionTagPrefix) ? latestVersion.split(versionTagPrefix)[1] : latestVersion;
```

Comment:
Looks like this could be replaced with `const semanticVersion = this.state.latestReleaseVersion.replace(/^v/);`, correct?